packages/hash/frontend/src/blocks/page/createPlaceholderPlugin/Placeholder.tsx

```diff
@@ -0,0 +1,28 @@
+import { styled, Typography } from "@mui/material";
+
+const Command = styled(Typography)(({ theme }) => ({
+  color: theme.palette.gray[60],
+  borderRadius: 3,
+  background: theme.palette.gray[10],
+  width: 30,
+  height: 30,
```

Comment:
Similar comment as the container `lineHeight` - can we make these numbers relative to the `lineHeight`, and make that relative to the paragraph styling, so that we can update a single number and have everything else follow?

Reply:
Should be solved https://github.com/hashintel/hash/pull/953/commits/ed4052c88825c5a0e07fe5683de4f3c9985416eb